build/scripts-3.4/pydeface.py

```diff
@@ -0,0 +1,180 @@
+#!/opt/local/bin/python
+"""
```

Comment:
this should probably be one of (depending on your preference):

```
#!/usr/bin/env python2
#!/usr/bin/env python3
#!/usr/bin/env python
```
Because likely the user is going to want to just execute, and then not have it installed at opt.